terraform/gcp/projects/pilot-hubs.tfvars

```diff
@@ -15,6 +15,14 @@ notebook_nodes = {
     max : 20,
     machine_type : "n1-highmem-4",
     labels: { }
+  },
+  "paleo": {
+    min: 0,
```

Comment:
A quick question - if the goal here is to make user sessions start up more quickly, wouldn't we want `min` to be something greater than 0? Otherwise what is the benefit that we get by defining a dedicated node pool? (I am happy to update some infrastructure docs somewhere if this isn't already documented)

Reply:
Yep, in the [referenced commit](https://github.com/2i2c-org/infrastructure/commit/20969eb31b62b9dd85cebc6080a6e8c516a3a047) the minimum size was actually 5.

Reply:
@choldgraf @damianavila ah, I usually set it to 0 and scale up / down manually when needed via the UI.

Reply:
> I usually set it to 0 and scale up / down manually when needed via the UI.

We should probably document that practice, though...